fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/PlanType.java

```diff
@@ -183,6 +183,7 @@ public enum PlanType {
     ALTER_CATALOG_RENAME_COMMAND,
     ALTER_ROLE_COMMAND,
     ALTER_VIEW_COMMAND,
+    ALTER_VIEW_COMMENT_COMMAND,
```

Comment:
maybe should like alter table, use one command to do both change query and change command